chris.py

```diff
@@ -11,6 +11,34 @@
 from timeutil import Weekday, weekly_event
 
 
+# Website is sorta static so it might be fine to just add the tanzcaffee this way: FIXME
+def create_tanzcaffee() -> list[DanceEvent]:
+    events = []
+    # Every day except saturday
+    for weekday in [
+        Weekday.MON,
+        Weekday.TUE,
+        Weekday.WED,
+        Weekday.THU,
+        Weekday.FRI,
+        Weekday.SUN,
+    ]:
+        events += weekly_event(
+            weekday,
+            DanceEvent(
+                starts_at=dateparser.parse("17:00"),
+                ends_at=dateparser.parse("18:00"),
+                name="Tanzcafe",
+                price_euro_cent=500,
+                description="5-Uhr-Tee in Wien",
```

Comment:
**thought:** The description could be longer as the website too has more information.

Reply:
Done